sonar-server/obs-gap-server.go

```diff
@@ -122,8 +122,8 @@ func (s *obsGapServer) getEventResourceNamespace(event map[string]interface{}) s
 	}
 }
 
-func (s *obsGapServer) isSameTarget(currentEvent map[string]interface{}) bool {
-	return s.getEventResourceName(currentEvent) == s.ceName && s.getEventResourceNamespace(currentEvent) == s.ceNamespace
+func isSameTarget(e1, e2 map[string]interface{}) bool {
```

Comment:
Please revert this change. `getEventResourceName(e1) == getEventResourceName(e2)` does not always work because the "crucial event" passed to the server might not contain the `name` or `namespace` field.